addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -94,6 +94,18 @@ $o-wslides-fs-side-width: 300px;
     .o_wslides_home_nav {
         top: -40px;
 
+        [style*="background: white"] .nav-link {
+            color: $navbar-light-color !important;
+
+            @include hover-focus {
+                color: $navbar-light-hover-color !important;
+            }
+
+            &.disabled {
+                color: $navbar-light-disabled-color !important;
+            }
+        }
```

Comment:
I guess this could be an acceptable fix in stable... I guess we would remove the inline style in master? If yes, I would mark a "TODO remove me in master" comment here and a "TODO remove inline style in master" comment next to the related XML definition.